rfcs/0002-typography-scaling.md

```diff
@@ -0,0 +1,162 @@
+---
+Start Date: 05-01-2021
+RFC PR: NA
+Blade Issue: NA
+---
+
+# Typography scaling across multiple devices <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Basic Example](#basic-example)
+  - [On Desktop](#on-desktop)
+  - [On Mobile](#on-mobile)
+- [Motivation](#motivation)
+  - [Why do we need a type scale?](#why-do-we-need-a-type-scale)
+  - [Use Cases](#use-cases)
+  - [Expected Outcome](#expected-outcome)
+- [Detailed Design](#detailed-design)
+  - [Our Approach](#our-approach)
+  - [Exception](#exception)
+  - [Example](#example)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+  - [Multiple type-scaling across breakpoints](#multiple-type-scaling-across-breakpoints)
+  - [No type-scaling across breakpoints](#no-type-scaling-across-breakpoints)
+  - [Fluid Typography](#fluid-typography)
+  - [Custom Scaling](#custom-scaling)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Typography is an integral part of web design. The user who is consuming information from our dashboard should not face any challenges while reading/analyzing data. Therefore we need to make sure that the type element should scale well across multiple devices (desktops/mobile/tablets/etc.).
+
+# Basic Example
+This is how the typography will look after the implementation of type scales.
+
+#### **On Desktop**
+<img width="600" src="./images/website-example-desktop.png" />
+<img width="600" src="./images/dashboard-example-desktop.png" />
+
+---
+
+#### **On Mobile**
+<div style="display:flex; align-items: flex-start">
+<img width="300" src="./images/website-example-mobile.png" />
+<img width="300" src="./images/dashboard-example-mobile.png" />
```

Comment:
The "Recent Activity" section on desktop is `20px` and on mobile its `18px`. It's scaling down?

Reply:
Yes! Because it is proportionate for smaller devices (like mobile).